libraries/AP_HAL_ChibiOS/hwdef/DAKEFPVF405/README.md

```diff
@@ -0,0 +1,105 @@
+# DAKEFPV F405 Flight Controller
+
+The DAKEFPV F405 is a flight controller produced by [DAKEFPV](https://www.dakefpv.com/).
+
+## Features
+
+ - MCU - STM32F405 32-bit processor. 1024Kbytes Flash
+ - IMU - Dual ICM42688
+ - Barometer - SPL06
+ - OSD - AT7456E
+ - Onboard Flash: 16MByte
+ - 8x UARTs
+ - 13x PWM Outputs (12 Motor Output, 1 LED)
+ - Battery input voltage: 4S-12S
+ - BEC 3.3V 0.5A
+ - BEC 5V 3A
+ - BEC 12V 3A for video, gpio controlled
+ - Dual switchable camera inputs
+
+## Pinout
+
+![DAKEFPV F405 Board Top](Top.jpg "DAKEFPV F405 Top")
+![DAKEFPV F405 Board Bottom](Bottom.jpg "DAKEFPV F405 Bottom")
+
+## DAKEFPV F405 Wiring Diagram
+
+![DAKEFPV F405 Wiring Diagram Top]( WiringDiagramTop.jpg "DAKEFPV F405 Wiring Diagram Top")
+![DAKEFPV F405 Wiring Diagram Bottom](WiringDiagramBottom.jpg "DAKEFPV F405 Wiring Diagram Bottom")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (GPS, DMA-enabled)
+ - SERIAL2 -> UART2 (ELRS)
```

Comment:
```suggestion
 - SERIAL2 -> UART2 (RCin) RX DMA capable
```
RX DMA is required for a UART used for RCinput on CRSF/ELRS for reliability, so its good it has it

Reply:
Corrected as required.